src/dymaptic.GeoBlazor.Core/Scripts/arcGisJsInterop.ts

```diff
@@ -2112,7 +2113,17 @@ export async function createLayer(layerObject: any, wrap?: boolean | null, viewI
             newLayer = kmlLayer;
             copyValuesIfExists(layerObject, kmlLayer, 'sublayers', 'blendMode', 'maxScale', 'minScale', 'title', 'visible');
             break;
-        default:
+        case 'wcs':
+            newLayer = new WCSLayer({
+                url: layerObject.url,
+                renderer: layerObject.renderer,
+                multidimensionalDefinition: layerObject.multidimensionalDefinition
+            });
+            let wcsLayer = newLayer as WCSLayer;
+            newLayer = wcsLayer;
+            copyValuesIfExists(layerObject, wcsLayer, 'bandIds', 'copyright', 'coverageId', 'coverageInfo', 'customParameters', 'fields', 'interpolation', 'maxScale', 'minscale', 'multidimensionalDefinition', 'rasterInfo', 'renderer');
```

Comment:
You may need to use the jsBuilder here rather then the copy values for the render and mulidimensionDefinition